framework/src/main/java/org/tron/common/logsfilter/capsule/TransactionLogTriggerCapsule.java

```diff
@@ -156,6 +163,100 @@ public TransactionLogTriggerCapsule(TransactionCapsule trxCapsule, BlockCapsule
+                      .encode58Check(unfreezeBalanceContract.getReceiverAddress().toByteArray()));
+                }
+              }
+              break;
+            case FreezeBalanceV2Contract:
+              FreezeBalanceV2Contract freezeBalanceV2Contract = contractParameter
+                  .unpack(FreezeBalanceV2Contract.class);
+
+              if (Objects.nonNull(freezeBalanceV2Contract)) {
+                if (Objects.nonNull(freezeBalanceV2Contract.getOwnerAddress())) {
+                  transactionLogTrigger.setFromAddress(StringUtil
+                      .encode58Check(freezeBalanceV2Contract.getOwnerAddress().toByteArray()));
+                }
+                transactionLogTrigger.setAssetAmount(freezeBalanceV2Contract.getFrozenBalance());
+              }
+              break;
+            case UnfreezeBalanceV2Contract:
+              UnfreezeBalanceV2Contract unfreezeBalanceV2Contract = contractParameter
+                  .unpack(UnfreezeBalanceV2Contract.class);
+
+              if (Objects.nonNull(unfreezeBalanceV2Contract)) {
+                if (Objects.nonNull(unfreezeBalanceV2Contract.getOwnerAddress())) {
+                  transactionLogTrigger.setFromAddress(StringUtil
+                      .encode58Check(unfreezeBalanceV2Contract.getOwnerAddress().toByteArray()));
+                }
+                transactionLogTrigger.setAssetAmount(
+                    unfreezeBalanceV2Contract.getUnfreezeBalance());
+              }
+              break;
+            case WithdrawExpireUnfreezeContract:
+              WithdrawExpireUnfreezeContract withdrawExpireUnfreezeContract = contractParameter
+                  .unpack(WithdrawExpireUnfreezeContract.class);
+
+              if (Objects.nonNull(withdrawExpireUnfreezeContract)
+                  && Objects.nonNull(withdrawExpireUnfreezeContract.getOwnerAddress())) {
+                transactionLogTrigger.setFromAddress(StringUtil.encode58Check(
+                      withdrawExpireUnfreezeContract.getOwnerAddress().toByteArray()));
+              }
+              break;
+            case DelegateResourceContract:
+              DelegateResourceContract delegateResourceContract = contractParameter
+                  .unpack(DelegateResourceContract.class);
+
+              if (Objects.nonNull(delegateResourceContract)) {
+                if (Objects.nonNull(delegateResourceContract.getOwnerAddress())) {
+                  transactionLogTrigger.setFromAddress(StringUtil
+                      .encode58Check(delegateResourceContract.getOwnerAddress().toByteArray()));
+                }
+                if (Objects.nonNull(delegateResourceContract.getReceiverAddress())) {
+                  transactionLogTrigger.setToAddress(StringUtil
+                      .encode58Check(delegateResourceContract.getReceiverAddress().toByteArray()));
+                }
+                transactionLogTrigger.setAssetAmount(
```

Comment:
`transactionLogTrigger.setAssetName("trx");`?

Reply:
OK，I get it.